backend/pkg/metrics/prometheus_metrics.go

```diff
@@ -0,0 +1,158 @@
+package metrics
+
+import (
+	"fmt"
+	"github.com/go-openapi/strfmt"
+	"github.com/openclarity/kubeclarity/api/server/restapi/operations"
+	"github.com/openclarity/kubeclarity/backend/pkg/database"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+	"net/http"
+	"strings"
+	"time"
+)
+
+const (
+	// Prefix is to make metrics unique
+	Prefix string = "kubeclarity"
+)
+
+var (
+	applicationCounter = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_applications",
+		Help: "The total number of applications",
+	})
+	resourceCounter = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_resources",
+		Help: "The total number of resources",
+	})
+	packageCounter = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_packages",
+		Help: "The total number of packages",
+	})
+	vulnerabilityCounter = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_vulnerabilities_total",
+		Help: "The total number of vulnerabilities",
+	})
+	fixableVulnerabilityCounter = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_fixable_vulnerabilities_total",
+		Help: "The total number of fixable vulnerabilities",
+	})
+	fixableVulnerability = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_fixable_vulnerabilities",
+		Help: "The number of fixable vulnerabilities per severity",
+	}, []string{"severity"})
+	vulnerability = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: Prefix + "_number_of_vulnerabilities",
+		Help: "The number of vulnerabilities per severity",
+	}, []string{"severity"})
+	trendGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: Prefix + "_vulnerability_trend",
+		Help: "Incoming vulnerabilities within the last hour",
+	}, []string{"severity"})
```

Comment:
Missing a documentation in the README.md.
Also, not sure about the description here, what is the meaning of `Incoming vulnerabilities`?
trends are used in the dashboard to show new vulnerabilities (to that system) that was found in a time period.